### DISCUSSION TITLE
poc verify between vim and nvim

### DISCUSSION THREAD

**Discussion by xiaoqieding** [vim/vim#15560] [Q&A] [answered]
I'm doing poc verify on vim which Out-of-bounds write in function append_command.
the link is https://huntr.com/bounties/f61a64e2-d163-461b-a77e-46ab38e021f0
And it works.
Meanwhile i found that the same poc on nvim isn't works though the part of the code is really the same.
nvim's patch is 664efa497e4e3d79d2e4ab486acbf1471b2389b0
and vim patch is bfaa24f95343af9c058696644375d04e660f1b00
maybe there is someone who can help solve this question .Thanks

**Comment by chrisbra** [comment]
are you saying the poc still causes an buffer overflow in Vim? The Vim patch that fixes it is 44a3f3353e0407e9fffee138125a6927d1c9e7e5

Please note, if you are reporting a potential security relevant issue, please use Githubs Security Advisory Feature and do not disclose this in public.

**Reply by xiaoqieding** [reply, marked as answer]
no，what i mean is the same poc and the same code in vim and nvim cause different results.
it's not a security-related issue，just an experimental results.
